Source/JavaScriptCore/llint/InPlaceInterpreter64.asm

```diff
@@ -5756,12 +5772,46 @@ end
     # store entrypoint and target instance on the stack for now
     push r0, r1
 
-    # determine new stack pointer
+    #  entrypoint, targetInstance  <- sp
+    #  argument n                  <- sc1
+    #  argument n-1
+    #  ...
+    #  argument 0
+    #  (IPInt stack)
+    #  (IPInt locals)
+    #  call frame                  <- cfr
+    #  call frame
+    #  argument
+    #  argument
+    #  argument
+    #  argument
+    #  return val
+    #  return val
+    #  <caller frame>
```

Comment:
In the rest of JSC we have the caller's frame textually above the callee's frame as the stack grows down in memory. I think we should be consistent here.